src/main/java/org/jabref/gui/libraryproperties/LibraryPropertiesDialogViewModel.java

```diff
@@ -59,8 +62,15 @@ public LibraryPropertiesDialogViewModel(BasePanel panel, DialogService dialogSer
         Optional<String> fileDI = metaData.getUserFileDirectory(preferencesService.getUser());
         fileDI.ifPresent(userSpecificFileDirectoryProperty::setValue);
 
+        metaData.getLaTexFileDirectory(preferencesService.getUser()).map(Path::toString).ifPresent(laTexFileDirectoryProperty::setValue);
+        
+        Optional<Path> fileDL = metaData.getLaTexFileDirectory(preferencesService.getUser());
```

Comment:
These three lines can now be deleted, right?

Reply:
Oh sorry. I thought I had deleted the lines. Thank you for pointing it out